app/routes/post.py

```diff
@@ -49,6 +49,16 @@ def get_post_preview(id):
     return render_template('post/preview.html', id=id)
 
 
+@server.route("/answer/<int:answer_id>/")
+def get_answer(answer_id):
+    answer = answer_controller.get_answer(answer_id)
+    try:
+        page = answer_controller.get_page(answer)
+    except ValueError:
+        return abort(404)
+    return redirect(url_for('get_post', post_id=answer.post_id, p=page) + f"#answer-{answer.id}", code=301)
```

Comment:
we might want to add `title=slugify(post.title)` but not 100% necessary since the `/post/:id` will redirect anyway.

Reply:
Yeah I left that out because a) that would be another DB hit to get the post title, and b) the post route will get it anyway.